lsm-tree.cabal

```diff
@@ -541,14 +542,6 @@ library rocksdb
     , bytestring
     , indexed-traversable
 
-library monkey
-  import:          language, warnings, wno-x-partial
-  hs-source-dirs:  src-monkey
-  exposed-modules: Monkey
-  build-depends:
-    , ad    ^>=4.5.6
```

Comment:
FWIW, my plan was to drop using `ad` as we settle on the merging policy. As currently implemented, the policy is in `Monkey.hs` is wrong (the last level is not leveling, it's also tiering).

Also, I'd keep the calculations as a separate library. We are not folding in `bloomfilter` not `xxhash` into `lsm-tree` either. Also `monkey` calculations are standalone and complete set of stuff (though less useful standalone)

Reply:
Alright, reverted. I moved some functions in to the `monkey` library instead of the other way around